fix: use updateProductVariantPrices mutation to save variant price

trojanh opened the pull request:
Signed-off-by: trojanh <rajan.tiwari@kiprosh.com>

Resolves https://github.com/reactioncommerce/reaction-admin/issues/170
Impact: **breaking|major**  
Type: **bugfix**

## Issue
Unable to set product variant price
1. Create a new variant.
2. Add a price to it
3. Refresh to see that the price is reset to zero.

## Solution
Use  `updateProductVariantPrices` mutation to set the price.

## Breaking changes
none

## Testing
1. Create a new variant.
2. Add a price to it
3. Refresh to see that the price is NOT reset to zero.
4. Also publish and refresh the variant to see it the changes get saved.

![variant-price-update](https://user-images.githubusercontent.com/22974490/71871561-873f6180-313f-11ea-8fd8-b1e03f8dcee9.gif)

trojanh committed:
fix: use updateProductVariantPrices mutation to save variant price

Signed-off-by: trojanh <rajan.tiwari@kiprosh.com>

spencern commented:
Thanks for the PR!

willopez commented on imports/plugins/included/product-admin/client/hocs/withVariantForm.js:
```diff
@@ -189,6 +199,28 @@ const wrapComponent = (Comp) => (
+          }
+        });
+      } catch (error) {
+        Alerts.toast(i18next.t("productDetailEdit.updateProductFieldFail", { [fieldName]: value }), "error");
```
For new code we want to use the new notification library `notistack` as seen here: https://github.com/reactioncommerce/reaction-admin/blob/e9cc890e3a1b63cec0804b260a161ed60f685a79/imports/plugins/core/taxes/client/containers/GeneralTaxSettings.js#L31 Please update to use it.

willopez reviewed (changes requested):
One small change necesary, otherwise looks good.

manueldelreal commented:
Save for @willopez review comment and the linting errors on the checks this one does permit the price update via the admin ui.

trojanh committed:
Add snackbar in withVariantForm HOC

Signed-off-by: trojanh <rajan.tiwari@kiprosh.com>

trojanh committed:
Merge branch 'trunk' of github.com:/reactioncommerce/reaction-admin into set-variant-price

Signed-off-by: trojanh <rajan.tiwari@kiprosh.com>

trojanh commented:
@willopez I have replaced all occurrences of `Alerts.alert` and `Alert.toast` to use `snackbar`. Also, fixed the linting error and rebased the branch. Please review the changes.

willopez commented:
thanks @trojanh!
